articles/onboarding/appliance-sprint.md

```diff
@@ -34,7 +34,7 @@ Appliance Sprint is Auth0’s onboarding program for enterprise customers choosi
 
 **What?** Your team prepares the relevant infrastructure based on our guidance.
 
-Appliance Infrastructure Requirements Docs - Coming Soon!
+[Appliance Infrastructure Requirements Docs](https://auth0.com/docs/appliance/infrastructure) 
```

Comment:
Should be:
`[Appliance Infrastructure Requirements Docs](/appliance/infrastructure)`

We dont use absolute links on the docs site.